Dockerfile

```diff
@@ -1,13 +1,17 @@
-FROM nfcore/base:1.9
+FROM nfcore/base:1.10.2
 LABEL authors="Leon Bichmann, Lukas Heumos, Alexander Peltzer" \
-      description="Docker image containing all requirements for nf-core/mhcquant pipeline"
+      description="Docker image containing all software requirements for the nf-core/mhcquant pipeline"
 
 # Install the conda environment
 COPY environment.yml /
-RUN conda env create -f /environment.yml && conda clean -a
+RUN conda env create --quiet -f /environment.yml && conda clean -a
+
+# Add conda installation dir to PATH (instead of doing 'conda activate')
+ENV PATH /opt/conda/envs/nf-core-mhcquant-dev/bin:$PATH
 
 # Dump the details of the installed packages to a file for posterity
-RUN conda env export --name nf-core-mhcquant-1.5.1 > nf-core-mhcquant-1.5.1.yml
+RUN conda env export --name nf-core-mhcquant-dev > nf-core-mhcquant-dev.yml
```

Comment:
Normally we do bump versions to `1.5.2dev` for example, but as you want to release soon again it is ok like that for now